### PR TITLE
feat:added login screen and passcode

### LINKED ISSUE #3: Adding login and passcode view
**Summary:**

Adding login and passcode feature in the app.

### PR DESCRIPTION
**Pull request by ivary43**
Fixes #3 

Please Add Screenshots If there are any UI changes.

Please make sure these boxes are checked before submitting your pull request - thanks!

- \[x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- \[x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by ivary43** [comment]
@therajanmaurya please review.

File: app/src/main/kotlin/org/mifos/mobile/cn/ui/mifos/login/LoginActivity.kt [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+import org.mifos.mobile.cn.ui.mifos.passcode.PasscodeActivity
+import org.mifos.mobile.cn.ui.utils.ConstantKeys
+import org.mifos.mobile.cn.ui.utils.Toaster
+import javax.inject.Inject
+
+
+class LoginActivity : MifosBaseActivity(), LoginContract.View {
+
+
+    @BindView(R.id.et_username)
```
Comment (therajanmaurya) [inline app/src/main/kotlin/org/mifos/mobile/cn/ui/mifos/login/LoginActivity.kt]: We are using kotlin extenstion, you can access ui instance using their ids

File: app/src/main/kotlin/org/mifos/mobile/cn/ui/mifos/login/LoginActivity.kt [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+        //TODO:remove this while implementing API
+        preferencesHelper.putUsername("fineractCn")
+        preferencesHelper.putPassword("password")
+    }
+
+    @OnClick(R.id.btn_login)
+    internal fun login() {
+
+        val username = et_Username.text.toString().trim()
```
Comment (therajanmaurya) [inline app/src/main/kotlin/org/mifos/mobile/cn/ui/mifos/login/LoginActivity.kt]: Use camel case for naming convention like etUserName

File: app/src/main/kotlin/org/mifos/mobile/cn/ui/mifos/login/LoginActivity.kt [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+        finish()
+    }
+
+    override fun showError(errorMessage: String) {
+        Toaster.show(findViewById(android.R.id.content), errorMessage, Toaster.SHORT)
+    }
+
+    override fun showProgress() {
+        showProgressbar()
+        Toaster.show(findViewById(android.R.id.content), getString(R.string.logging_in), Toaster.SHORT)
```
Comment (therajanmaurya) [inline app/src/main/kotlin/org/mifos/mobile/cn/ui/mifos/login/LoginActivity.kt]: why toast is here.

**Pull request merged**